shell.c

```diff
@@ -306,16 +309,94 @@ char* readLine() {
     }
 }
 
+// Display result of the assignment
+struct pre_assessment {
+    bool valid;
+    char *assignment;
+};
+
+void display_assignment_result(struct pre_assessment pa, char *line) {
+    // Retrieve the first 8 characters forming the "echo -e "
+    const int COMM_SIZE = 8;
```

Comment:
Constant values should be defined at the top.

Reply:
I am still testing thus I will move it up once it is done.